### PR TITLE
usbl merge please

### PR DESCRIPTION
**Pull request by softbalanced**

### CODE REVIEW

**Review by smk762 — COMMENTED** [review]
Needs one more change - 
add `"USBL": "#279553" ` to atomic_defi_design/qml/Constants/Style.qml

File: assets/config/0.5.1-coins.json [hunk 1]
```diff
@@ -5266,6 +5266,30 @@
     "active": false,
     "currently_enabled": false
   },
+  "USBL": {
+    "active": true,
+    "coin": "USBL",
+    "name": "Balanced",
+    "coingecko_id": "balanced",
+    "coinpaprika_id": "balanced-balanced",
+    "currently_enabled": true,
```
Comment (smk762) [inline assets/config/0.5.1-coins.json:5275]: change to false

File: assets/config/0.5.1-coins.json [hunk 2]
```diff
@@ -5266,6 +5266,30 @@
     "active": false,
     "currently_enabled": false
   },
+  "USBL": {
+    "active": true,
```
Comment (smk762) [inline assets/config/0.5.1-coins.json:5270]: change to false

File: assets/config/0.5.1-coins.json (hunk 1 above)
Comment (softbalanced) [reply assets/config/0.5.1-coins.json]: done

File: assets/config/0.5.1-coins.json (hunk 2 above)
Comment (softbalanced) [reply assets/config/0.5.1-coins.json]: done

File: atomic_defi_design/qml/Constants/Style.qml [hunk 3]
```diff
@@ -440,5 +440,6 @@ QtObject {
                                           "ZIL": "#42BBB9",
                                           "ZRX": "#302C2C",
                                           "UNI": "#FF007A"
```
Comment (cipig) [inline atomic_defi_design/qml/Constants/Style.qml:442]: "," is missing on this line

Comment (softbalanced) [reply atomic_defi_design/qml/Constants/Style.qml]: updated

**Review by smk762 — APPROVED** [review]

File: assets/config/0.5.1-coins.json [hunk 4]
```diff
@@ -5266,6 +5266,30 @@
     "active": false,
     "currently_enabled": false
   },
+  "USBL": {
+    "active": false,
+    "coin": "USBL",
+    "name": "Balanced",
+    "coingecko_id": "balanced",
+    "coinpaprika_id": "balanced-balanced",
```
Comment (cipig) [inline assets/config/0.5.1-coins.json:5274]: @softbalanced this ids don't exist on coingecko/coinpaprika

**Pull request merged**